src/renderer/components/content/editor/editor.js

```diff
@@ -85,6 +90,10 @@ var Editor = React.createClass({
   },
 
   render: function() {
+    var deleteBtn = null;
+    if (this.props.note.get('_id')) {
+      deleteBtn = <button onClick={this.deleteNote}>DELETE</button>;
```

Comment:
Can the text for this button be Delete (rather than DELETE), to be consistent with the other buttons?